app/modules/intelligence/agents/agents/debug_rag_agent.py

```
@@ -10,10 +10,15 @@
 
 from app.modules.code_provider.code_provider_service import CodeProviderService
 from app.modules.conversations.message.message_schema import NodeContext
-from app.modules.intelligence.provider.provider_service import (
-    AgentType,
-    ProviderService,
+from app.modules.intelligence.llm_provider.llm_provider_service import (
+    LLMProviderService,
 )
+from app.modules.intelligence.prompts_provider.agent_prompts_provider import (
+    AgentPromptsProvider,
```

Ruff (0.8.2)

`app.modules.intelligence.prompts_provider.agent_prompts_provider.AgentPromptsProvider` imported but unused

Remove unused import: `app.modules.intelligence.prompts_provider.agent_prompts_provider.AgentPromptsProvider`

(F401)